packages/contracts/test/voting/token-voting.ts

```diff
@@ -463,6 +489,23 @@ describe('TokenVoting', function () {
   });
 
   describe('Proposal + Execute:', async () => {
+    beforeEach(async () => {
+      const receivers = Array.from(Array(12).keys()).map(
```

Comment:
either the `slice` option or the proposed helper

Reply:
Fixed in 26e42329bc5434e96fd75c0bb61139616678b262